CC/Sounder/Radio.cc

```diff
@@ -6,6 +6,10 @@
 
 void Radio::dev_init(Config* _cfg, int ch, double rxgain, double txgain)
 {
+    // Set sampling rate
```

Comment:
Interesting find! I am still not sure why this causes the change in behavior. The reason I set the sample rate in the init, was that it essentially turns on the radio logic in the FPGA before the IQ calibration step (which is disabled by default).

Can we check this in on a different branch and do a separate PR?